framework/include/utils/MooseUtils.h

```diff
@@ -110,6 +110,12 @@ void serialEnd(const libMesh::Parallel::Communicator & comm);
  */
 bool hasExtension(const std::string & filename, std::string ext, bool strip_exodus_ext = false);
 
+/**
+ * Removes any file extension from the fiven string s (i.e. any ".[extension]" suffix of s) and
+ * returns the result.
+ */
+std::string stripExtension(const std::string & s);
```

Comment:
https://github.com/idaholab/moose/blob/devel/framework/include/utils/MooseUtils.h#L120

Reply:
That doesn't strip extensions - it splits file name from prefix dirpaths.

Reply:
So the method called stripExtensions doesn't actually strip extensions 😛  ?!

Reply:
No - the function you linked (``splitFileName``) doesn't strip extensions.

Reply:
👍